tests/components/zwave_js/test_switch.py

```diff
@@ -0,0 +1,85 @@
+"""Test the Z-Wave JS sensor platform."""
```

Comment:
Should probably say "test the Z-Wave JS switch platform"

Reply:
Good catch! Opened a PR to fix #45076